clients/unshaded/src/main/java/tachyon/client/BlockOutStream.java

```diff
@@ -58,12 +58,14 @@ public static BlockOutStream get(TachyonFile tachyonFile, WriteType opType, int
    */
   public static BlockOutStream get(TachyonFile tachyonFile, WriteType opType, int blockIndex,
       long initialBytes, TachyonConf tachyonConf) throws IOException {
-
-    if (tachyonFile.mTachyonFS.hasLocalWorker()
-        && tachyonConf.getBoolean(Constants.USER_ENABLE_LOCAL_WRITE,
-            Constants.DEFAULT_USER_ENABLE_LOCAL_WRITE)) {
+    if ((tachyonFile.mTachyonFS.hasLocalWorker()
+            && tachyonConf.getBoolean(Constants.USER_ENABLE_LOCAL_WRITE,
+            Constants.DEFAULT_USER_ENABLE_LOCAL_WRITE)
+            && tachyonConf.get(
```

Comment:
To make it in a proper way, we should use the worker chosen for the condition that is to say
- if the worker is local then write with local stream and
- if the worker is remote then write with remote stream. 
The problem is how to know the chosen worker at this level ?